### PR TITLE
Typo in Basic Data Structures: Combine Arrays with the Spread Operator #18003 FIX

### PR DESCRIPTION
**Pull request by TomerPacific**
#### Description

Fixing typo in algorithms and data structures lesson having to do with combining arrays with the spread operator. Deleted the multiple word 'more'.

#### Pre-Submission Checklist

- [X] Your pull request targets the `dev` branch.
- [X] Branch starts with either `fix/`, `feature/`, or `translate/` (e.g. `fix/challenge-tests`)
- [X] All new and existing tests pass the command `npm test`.
- [X] Use `npm run commit` to generate a conventional commit message.
    Learn more here: <https://conventionalcommits.org/#why-use-conventional-commits>
- [X] The changes were done locally on your machine and NOT GitHub web interface.
    If they were done on the web interface you have ensured that you are creating conventional commit messages.

#### Checklist:

- [X] Tested changes locally.
- [X] Addressed currently open issue (replace XXXXX with an issue no in next line)

Closes https://github.com/freeCodeCamp/freeCodeCamp/issues/18003

### CODE REVIEW

**Comment by Nirajn2311** [comment]
@TomerPacific in your PR description, when you put a ‘x’ in [] there should be no spaces otherwise it won’t be recognised as a checkbox.

Also before you commit your changes, execute the command `npm run commit` as FCC follows a certain commit message structure and Travis CI follows it otherwise the build will fail.

**Comment by TomerPacific** [comment]
@Nirajn2311 ,
thank you for your feedback.

Do I just close the pull request now?

**Comment by Nirajn2311** [comment]
@TomerPacific you have to wait until one of FCC main guys checks this PR and merges it. Then the PR will automatically close and you will have to close the issue back in FCC repo.

**Comment by scissorsneedfoodtoo** [comment]
@TomerPacific, unfortunately it seems like the Travis CI build is failing. It's an easy fix, though, since it's just a problem with your commit message.

Please take a look at the [Commit Your Changes](https://github.com/freeCodeCamp/curriculum/blob/dev/docs/CONTRIBUTING.md#commit-your-changes) section of the Contributor's Guide. There are some instructions about conventional commit messages and how to edit your message there.

Basically you want to change the current message from `Fixing typo in algorithms and data structures lesson having to do with combining arrays with the spread operator. Deleted the multiple more word` to something like `fix(challenges): fixed typo in algorithms and ds` by running `git commit --amend` on your `feature/fixing-typo-spread-operator` branch. Then you should be good to go after a force push back to your remote branch.

**Comment by TomerPacific** [comment]
Hey @scissorsneedfoodtoo ,
I have done as you have requested. Hopefully, the Travis CI build will pass and everything will be good to go.

**Comment by Nirajn2311** [comment]
@TomerPacific congrats on your first PR.

**Comment by scissorsneedfoodtoo** [comment]
:sparkles: :sparkles: :sparkles: Awesome! Congratulations on your first contribution to freeCodeCamp! :sparkles: :sparkles: :sparkles:

Thank you for submitting a PR for this issue, @TomerPacific. We're all really looking forward to your next contribution!
